web/src/whatwg-fetch.d.ts

```diff
@@ -0,0 +1 @@
+declare module "whatwg-fetch";
```

Comment:
sometimes typescript just wears me out. i don't even know what this does. i just want to polyfill fetch in the test. other suggestions accepted.

Reply:
I imagine `whatwg-fetch` doesn't ship with type definitions, I think this is fine since we're just using it for tests. What errors were you seeing?

Reply:
https://www.npmjs.com/package/@types/whatwg-fetch says "fetch types are now provided by '--lib dom'"

Reply:
that seems to mean i need tsconfig compileroptions: [dom]. but i already had that and still got warnings.

Reply:
Ok np